docs/handbook/image-file-formats.rst

```diff
@@ -31,6 +31,14 @@ The :py:meth:`~PIL.Image.Image.open` method sets the following
 **compression**
     Set to ``bmp_rle`` if the file is run-length encoded.
 
+DIB
+^^^
+
+PIL reads and writes DIB files. DIB files are similar to BMP files, so see
```

Comment:
Hmm, maybe we should replace PIL with Pillow in this file?

Reply:
Okay, done, with the exception of two references to PIL versions - https://github.com/python-pillow/Pillow/blame/master/docs/handbook/image-file-formats.rst#L1139-L1142